src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -417,7 +420,6 @@ export class InstructorCourseEditPageComponent implements OnInit {
       this.statusMessageService.showErrorToast(resp.error.message);
     });
 
-    panelDetail.editPanel.isEditing = false;
     panelDetail.originalPanel = JSON.parse(JSON.stringify(panelDetail.editPanel));
```

Comment:
Actually while it is not directly related to this bug, you can also move this line into `subscribe` to prevent invalid response from overwriting the original.